spec/factories/application_qualification.rb

```diff
@@ -3,6 +3,7 @@
     application_form
     level { %w[degree gcse other].sample }
     qualification_type { %w[BA Masters A-Level gcse].sample }
+    qualification_level { %w[Bachelor Master Foundation Doctorate].sample }
```

Comment:
I wonder if we should leave the factory till the next pr as usually we shouldn't be adding code that accesses new fields in the same pr (one on the checklist)

Reply:
Yes, agreed. I'll remove from this PR.